### ISSUE TITLE
Makefile appears to not be compatible with FreeBSD base make

### ISSUE THREAD

**Issue by eriknstr** [0x09/hfsfuse#1]
`freebsd-version`

```
11.0-RELEASE-p7
```

`make config`

```
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 10: Variable/Value missing from "export"
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 11: Variable/Value missing from "export"
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 16: Missing dependency operator
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 19: Missing dependency operator
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 20: warning: duplicate script for target "ifeq" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 18: warning: using previous script for "ifeq" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 20: warning: duplicate script for target "(,)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 18: warning: using previous script for "(,)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 21: warning: duplicate script for target "ifeq" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 18: warning: using previous script for "ifeq" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 21: warning: duplicate script for target "(,)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 18: warning: using previous script for "(,)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 22: Need an operator
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 31: Missing dependency operator
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 38: Need an operator
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 39: Missing dependency operator
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 40: warning: duplicate script for target "ifneq" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "ifneq" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 40: warning: duplicate script for target "(local,)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(local,)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 40: warning: duplicate script for target "(none)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(none)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 41: warning: duplicate script for target "ifneq" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "ifneq" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 41: warning: duplicate script for target "(local,)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(local,)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 41: warning: duplicate script for target "(none)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(none)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 42: warning: duplicate script for target "ifneq" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "ifneq" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 42: warning: duplicate script for target "(local,)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(local,)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 42: warning: duplicate script for target "(none)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(none)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 43: warning: duplicate script for target "ifneq" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "ifneq" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 43: warning: duplicate script for target "(local,)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(local,)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 43: warning: duplicate script for target "(none)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(none)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 44: warning: duplicate script for target "ifneq" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "ifneq" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 44: warning: duplicate script for target "(local,)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(local,)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 44: warning: duplicate script for target "(none)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(none)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 45: warning: duplicate script for target "ifneq" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "ifneq" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 45: warning: duplicate script for target "(local,)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(local,)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 45: warning: duplicate script for target "(none)" ignored
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 37: warning: using previous script for "(none)" defined here
make: "/usr/home/erikn/src/github.com/0x09/hfsfuse/Makefile" line 46: Need an operator
make: Fatal errors encountered -- cannot continue
make: stopped in /usr/home/erikn/src/github.com/0x09/hfsfuse
```

`gmake config`

```
echo CC=cc > config.mak
echo CONFIG_CFLAGS=-O3 -std=gnu11  >> config.mak
echo WITH_UBLIO=local >> config.mak
echo WITH_UTF8PROC=local >> config.mak
```

**Comment by 0x09** [comment]
Correct, requires gmake. I'll make a note about that.

**Comment by 0x09** [comment]
Closed with 5e2360d7ab728006d7aa11f45a24db13babeecad

So many things require gmake to be installed that I don't feel like it's too much baggage vs maintaining separate makefiles or getting automake involved.

**Comment by eriknstr** [comment]
For sure I agree, a simple note in the README is sufficient.
